_includes/footer.html

```diff
@@ -20,66 +21,72 @@ <h6 class="footerHeaderStyle align-center-md text-center-md">
-                          <li class="footerListStyle"><a href="{{ site.baseurl }}/school-administrators">For School Administrators</a></li>
-                          <li class="footerListStyle"><a href="{{ site.baseurl }}/parents">For Parents</a></li>
-                          <li class="footerListStyle"><a href="{{ site.baseurl }}/develop">For Developers</a></li>
-                          <li class="footerListStyle"><a href="{{ site.baseurl }}/sugar-stories">Sugar Stories</a></li>
-                          <li class="footerListStyle"><a href="{{ site.baseurl }}/music-blocks">Music Blocks</a></li>
-                          <li class="footerListStyle"><a href="{{ site.baseurl }}/turtle-blocks-js">Turtle Blocks JS</a></li>
-                        </ul>
-                      </article>
-                    </div><!-- We may want to move this to another page-->
-                    <div class="col-md-3 widget">
+
+                    <!-- Resources Section -->
+                    <div class="col-12 col-sm-6 col-md-3 widget mb-4 mb-md-0">
+                        <h4 class="footerHeaderStyle align-center-md"><b>Resources</b></h4>
+                        <article class="widget_content">
+                            <ul class="leftAlign align-center-col-md">
+                                <li class="footerListStyle"><a href="{{ site.baseurl }}/more">Culture and pedagogy</a></li><!--
+                                <li class="footerListStyle"><a href="http://schoolserver.org/">School Server Resources</a></li>
+                                <li class="footerListStyle"><a href="http://planet.sugarlabs.org">Planet Sugar Labs</a></li>--><!--
+                                <li class="footerListStyle"><a href="{{ site.baseurl }}/about-libre-software-culture">Libre Software Culture</a></li>
+                                <li class="footerListStyle"><a href="{{ site.baseurl }}/sugar-lesson-plans">Sugar Lesson Plans</a></li>-->
+                                <li class="footerListStyle"><a href="{{ site.baseurl }}/students">For Students</a></li>
+                                <li class="footerListStyle"><a href="{{ site.baseurl }}/school-administrators">For School Administrators</a></li>
+                                <li class="footerListStyle"><a href="{{ site.baseurl }}/parents">For Parents</a></li>
+                                <li class="footerListStyle"><a href="{{ site.baseurl }}/develop">For Developers</a></li>
+                                <li class="footerListStyle"><a href="{{ site.baseurl }}/sugar-stories">Sugar Stories</a></li>
+                                <li class="footerListStyle"><a href="{{ site.baseurl }}/music-blocks">Music Blocks</a></li>
+                                <li class="footerListStyle"><a href="{{ site.baseurl }}/turtle-blocks-js">Turtle Blocks JS</a></li>
+                            </ul>
+                        </article>
+                    </div>
```

Comment:
Unnecessary change in this div hides the actual change you are making when viewing diff; please remove unnecessary change.